src/core/exec-invoke.c

```diff
@@ -4016,6 +4019,25 @@ static int send_handoff_timestamp(
         return 1;
 }
 
+static int send_pidref(const ExecContext *c, ExecParameters *p, const PidRef *pidref, int *reterr_exit_status) {
+        int r;
+
+        assert(c);
+        assert(p);
+
+        if (p->pidref_fd < 0)
+                return 0;
+
+        r = send_one_fd_iov(p->pidref_fd, pidref->fd, &IOVEC_MAKE(&pidref->pid, sizeof(pidref->pid)), 1, 0);
```

Comment:
not your fault, but the naming here is quite confusing - maybe pidref_fd -> pidref_pipe or so?